src/to-be-made-a-dependency/foundation.ts

```diff
@@ -0,0 +1,104 @@
+import { get } from 'lit-translate';
+
+import { newOpenEvent } from '@openscd/open-scd-core';
+import {
+  COMPAS_SCL_PRIVATE_TYPE,
+  getCompasSclFileType,
+  getCompasSclName,
+  getPrivate,
+} from './compas-private.js';
+
+const FILE_EXTENSION_LENGTH = 3;
+
+export function getTypeFromDocName(docName: string): string {
```

Comment:
it will be used in compas-save, so we want to keep the whole foundation file as is, maybe in a later PR we can review again which functions are really not being used